cms/static/js/models/group.js

```diff
@@ -8,8 +8,17 @@ define([
             return {
                 name: '',
                 version: 1,
-                order: null
-             };
+                order: null,
+                showContentGroupUsages: false,
+                usage: []
+            };
+        },
+        url : function() {
+            return this.collection.parents[0].urlRoot + '/' + encodeURIComponent(this.collection.parents[0].id) + '/' + encodeURIComponent(this.id);
```

Comment:
There must be a parent for every group. I can't think of any case in which group exist without its parent.